server/channels/api4/cloud.go

```diff
@@ -140,7 +140,7 @@ func changeSubscription(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	var subscriptionChange *model.SubscriptionChange
-	if err = json.Unmarshal(bodyBytes, &subscriptionChange); err != nil {
+	if err = json.Unmarshal(bodyBytes, &subscriptionChange); len(bodyBytes) == 0 || err != nil {
```

Comment:
I don't think that approach helps here. If the input is `null`, ` len(bodyBytes)` will be greater then zero. Should we instead check if `subscriptionChange` is not longer `nil`?